fdbserver/DDTeamCollection.actor.cpp

```diff
@@ -3458,6 +3516,64 @@ Future<Void> DDTeamCollection::serverTeamRemover() {
 	return DDTeamCollectionImpl::serverTeamRemover(this);
 }
 
+Future<Void> DDTeamCollection::fixWrongReplicasLoop() {
+	return DDTeamCollectionImpl::fixWrongReplicasLoop(this);
+}
+
+void DDTeamCollection::fixWrongReplicas() {
```

Comment:
This function only fixes under-replication, right? We could call this `fixUnderReplicatedLargeTeams` or even just add a comment.